Filecontent_indexhtml and css/index.html

```diff
@@ -0,0 +1,275 @@
+	   </div>
+
+	   <div class="description_a_items ">
+		 <p class="company"> CANOPY </p>
+		 <div class="separator"></div>
+
+		 <p class="responsability"> Back End Dev </p>
+
+		 <div class="separator"></div>
+		 <p class="year"> 2015 </p>
+	   </div>
+
+	   <div class="projects_descriptions">
+		 <p> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+	   </div>
+
+	   <div class="programming_icon">
+
+		<a href="#" target="_blank" id="html1"><img src="Images/html.png"> </a>
+
+		<a href="#" target="_blank" id="html1"><img src="Images/css.png"> </a>
+		
+		<a href="#" target="_blank" id="html1"><img src="Images/javascript.png"> </a>  
+	  </div>
+
+	   </div>
+	   <button class="vizualization_projects"> See Project </button>
+	 </div>
+	 </section>
+
+          <section id="contains_work_section">
+          <div class="myself">
+        <div id="about-description-container" class="headline align-item">
+          <section id="about-description">
+            <h1 id="about-title" class="section-title">About <br>MySelf</h1>
+            <p class="headline-text"> Hello I'm a software developer! I can help you build a product , feature or website Look through some 
+              of my work and experience! 
+              If you like what you see and have a project you need coded, don't hestiate to contact me.
+            </p>
+          </section>
+
+
+			<div class="connect">
+
+			  <a href="#" target="_blank" id="button-connect"><img src="Images/LET'S CONNECT (1).png"> </a>
+
+			</div>
+			<ul class="Media-links">
+			  <li><a href="https://twitter.com/" target="_blank"><img src="./Images/twitter.png" alt="twitter icon"></a></li>
+			  <li><a href="https://www.linkedin.com/in/gardimy-charles-40b795212/" target="_blank"><img src="./Images/linkedin.png" alt="Linkedin icon"></a></li>
+			  <li><a href="#" target="_blank"><img src="./Images/medium.png" alt="medium icon"></a></li>
+			  <li><a href="https://github.com/Gardimy" target="_blank"><img src="./Images/github.png" alt="github icon"></a></li>
+			  <li><a href="#" target="_blank"><img src="./Images/Fingers icon.png" alt="Finger icon"></a></li>
+			   </ul>
+			   <div>
+			  <button class="download_resume"> Get my Resume </button>
+			  </div>
+
+			  <div class="all-skills-contains">
+				<ul>
+				  <li class="skills_title">
+					<p> Languages </p>
+					<div class="DW_Arrow"></div>
+				  </li>
+				  <li>
+					<ul class="skills-secondary">
+					  <li>
+						<img src="./Images/Js.icon.png">
+						<p> JavaScript </p>
+					  </li>
+					  <li>
+						<img src="./Images/Html.icon.png">
+						<p> HTML </p>
+					  </li>
+					  <li>
+						<img src="./Images/Css.icon.png">
+						<p> CSS </p>
+					  </li>
+					</ul>
+				  </li>
+				  <li class="skills-title list-closed">
+					<p> Frameworks </p>
+					<div class="right_arrow"></div>
+				  </li>
+				  <li class="skills-title list-closed">
+					<p> Skills </p>
```

Comment:
As shown in the figma design, Kindly show the dropdown and the right facing arrows that is to the right of the technology categories to match the figma design. :+1: 

Your design 

![Screenshot from 2022-12-10 23-34-19](https://user-images.githubusercontent.com/65233860/206879465-f3f1e81d-0a13-4884-9196-1c9a076efa7e.png)

The figma design 

![Screenshot from 2022-12-10 23-35-43](https://user-images.githubusercontent.com/65233860/206879487-39f4dd51-8862-4a94-8ca1-9f1b4c00979c.png)